src/Providers/SoapServiceProvider.php

```diff
@@ -8,13 +8,14 @@
 use RicorocksDigitalAgency\Soap\Request\Request;
 use RicorocksDigitalAgency\Soap\Request\SoapClientRequest;
 use RicorocksDigitalAgency\Soap\Soap;
+use RicorocksDigitalAgency\Soap\Support\Fakery\Fakery;
 
 class SoapServiceProvider extends ServiceProvider
 {
 
     public function register()
     {
-        $this->app->singleton('soap', fn() => app(Soap::class));
+        $this->app->singleton('soap', fn() => app(Soap::class, ['fakery' => app(Fakery::class)]));
```

Comment:
We shouldn't need the call to `app(Fakery::class)`

Reply:
Sorted! Thanks